sklearn/svm/tests/test_svm.py

```diff
@@ -764,6 +765,7 @@ def test_svc_nonfinite_params(global_random_seed):
         clf.fit(X, y)
 
 
+@pytest.mark.filterwarnings("ignore::FutureWarning")
 def test_unicode_kernel(global_random_seed):
```

Comment:
This test is really weird:

- "unicode" is the old name for the `str` class in Python 2, I think it refers to passing the name of a kernel rather than a callable;
- we alreay test this behavior many times elsewhere.
- it is just a smoke test and makes no assertions on the returned values.

I think we should just remove it as it's redundant with other tests.